downstream/assemblies/aap-hardening/assembly-hardening-aap.adoc

```diff
@@ -39,7 +39,7 @@ include::aap-hardening/ref-sensitive-variables-install-inventory.adoc[leveloffse
 //include::aap-hardening/ref-sudo-nopasswd.adoc[leveloffset=+3]
 include::aap-hardening/ref-initial-configuration.adoc[leveloffset=+1]
 include::aap-hardening/ref-infrastructure-as-code.adoc[leveloffset=+2]
-include::aap-hardening/con-controller-configuration.adoc[leveloffset=+2]
+//include::aap-hardening/con-controller-configuration.adoc[leveloffset=+2]
```

Comment:
In line 45 of this doc, let's change `include::aap-hardening/con-external-credential-vault.adoc[leveloffset=+3]` to `[leveloffset=+2]`, since we're removing `//include::aap-hardening/con-controller-configuration.adoc[leveloffset=+2]`